src/main/java/com/google/api/generator/spring/composer/comment/CommentComposer.java

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.spring.composer.comment;
+
+import com.google.api.generator.engine.ast.BlockComment;
+import com.google.api.generator.engine.ast.CommentStatement;
+import com.google.api.generator.engine.ast.LineComment;
+import com.google.api.generator.engine.ast.Statement;
+import java.util.Arrays;
+import java.util.List;
+
+public class CommentComposer {
+  protected static final String CLASS_HEADER_SUMMARY_PATTERN =
+      "Settings class to configure an instance of {@link %s}.";
```

Comment:
This seems to be the only difference from [`CommentComposer`](src/main/java/com/google/api/generator/gapic/composer/comment/CommentComposer.java)? If so, can we reuse gapic class and only have this addition in this new class?

Reply:
Removed class and moved relevant used properties to corresponding spring comment classes